Makefile

```diff
@@ -164,11 +164,6 @@ setup-and-test-cli: setup-test-infra venv ## Set up and run end to end tests
 test-container-dev: setup-test-infra venv ## CI ONLY Run container-structure-tests on locally built image
```

Comment:
nit: I think this could just be `test-container`

Reply:
sounds good. updated. thanks!